modules/sdk/gradle-plugins-poshi-runner/src/main/java/com/liferay/gradle/plugins/poshi/runner/PoshiRunnerExtension.java

```diff
@@ -71,6 +81,7 @@ public void setVersion(String version) {
 	private Object _baseDir = "poshi-tests";
 	private String _openCVVersion = "2.4.9-0.9";
 	private final Map<String, Object> _poshiProperties = new HashMap<>();
+	private Object _poshiRunnerProperties = "poshi-runner.properties";
```

Comment:
So are we going to require the file to be named poshi-runner.properties?  Is that the standard property file that poshi runner itself expects?